scripts/remove_downloads.py

```diff
@@ -0,0 +1,42 @@
+"""Remove all downloads.
+
+This script deletes all downloaded and unzipped files. It is
+useful for testing purposes. Needs to be updated with the directory
+structure or maybe can be deleted altogether later.
+"""
+
+import os
+
+# import click
+from faostat_data_primap.helper.definitions import downloaded_data_path
+
+
+def run():
```

Comment:
What is the delete function good for? Why would we want to delete version?

Reply:
We don't really need this script. I've used it to test the work flow. Once I have the tests set up I can delete it